app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -61,149 +62,217 @@ fun CreateActivityScreen(
-              value = placesLeft,
-              onValueChange = { placesLeft = it },
-              label = { Text("Places Left") },
-              modifier = Modifier.padding(8.dp).fillMaxWidth().testTag("inputPlacesCreate"),
-              placeholder = { Text("Places left/Total places") },
-          )
-          Spacer(modifier = Modifier.height(8.dp))
-          OutlinedTextField(
-              value = location,
-              onValueChange = { location = it },
-              label = { Text("Location") },
-              modifier = Modifier.padding(8.dp).fillMaxWidth().testTag("inputLocationCreate"),
-              placeholder = { Text("Where is it taking place") },
-          )
-          Spacer(modifier = Modifier.height(32.dp))
-          Button(
-              enabled = title.isNotEmpty() && description.isNotEmpty() && dueDate.isNotEmpty(),
-              onClick = {
-                val calendar = GregorianCalendar()
-                val parts = dueDate.split("/")
-                if (parts.size == 3) {
-                  try {
-                    calendar.set(
-                        parts[2].toInt(),
-                        parts[1].toInt() - 1, // Months are 0-based
-                        parts[0].toInt(),
-                        0,
-                        0,
-                        0)
-                     val activity =
-                        Activity(
-                            uid = listActivityViewModel.getNewUid(),
-                            title = title,
-                            description = description,
-                            date = Timestamp(calendar.time),
-                            price = price.toDouble(),
-                            placesLeft = parseFraction(placesLeft, 0)?.toLong() ?: 0.toLong(),
-                            maxPlaces = parseFraction(placesLeft, 1)?.toLong() ?: 0.toLong(),
-                            creator = creator,
-                            status = ActivityStatus.ACTIVE,
-                            location = location,
-                            images = carouselItems.map{it},
-                            participants = listOf()
+              Spacer(modifier = Modifier.height(8.dp))
+              OutlinedTextField(
+                  value = placesLeft,
+                  onValueChange = { placesLeft = it },
+                  label = { Text("Places Left") },
+                  modifier = Modifier.padding(8.dp).fillMaxWidth().testTag("inputPlacesCreate"),
+                  placeholder = { Text("Places left/Total places") },
+              )
+              Spacer(modifier = Modifier.height(8.dp))
+              OutlinedTextField(
+                  value = location,
+                  onValueChange = { location = it },
+                  label = { Text("Location") },
+                  modifier = Modifier.padding(8.dp).fillMaxWidth().testTag("inputLocationCreate"),
+                  placeholder = { Text("Where is it taking place") },
```

Comment:
we should add those strings in the xml constants file

Reply:
we need to have a uniform way of writing placeholders. here you ask a question, but in other equivalent placeholders, we rather give an instruction "Give the title" instead of asking "What is the title?"

that is a good reason to have all the strings in an xml file, to better see and edit this aspect of uniformity